### PR TITLE
Cekit build environment #158

### PR DESCRIPTION
**Pull request by rigazilla**
This enable the operator build with Cekit.
Just go under build/cekit and run:
`cekit build docker`

### CODE REVIEW

File: build/cekit/image.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+- name: "builder-infinispan-operator"
+  description: "Builder for Infinispan Operator"
+  version: latest
+  from: "fedora:29"
+  labels:
+    - name: "maintainer"
+      value: "jboss-support-dg6@redhat.com"
```
Comment (galderz) [inline build/cekit/image.yaml]: Most of this values are product specific. I'd leave those out for the community image and add them in the product overrides, is that possible?

File: build/cekit/image.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      value: "datagrid,java,jboss,xpaas,operator"
+    - name: "com.redhat.delivery.appregistry"
+      value: "true"
+  modules:
+    repositories:
+      - path: modules
+        name: modules
+    install:
+      - name: builder-infinispan-operator
+        #osbs:
```
Comment (galderz) [inline build/cekit/image.yaml]: You can remove commented sections from here. They should be added in product overrides.

File: build/cekit/image.yaml (hunk 1 above)
Comment (rigazilla) [reply build/cekit/image.yaml]: yes sure, done

File: build/cekit/image.yaml (hunk 2 above)
Comment (rigazilla) [reply build/cekit/image.yaml]: done. thanks @galderz !

**Pull request merged**